### DISCUSSION TITLE
[3.2] Preview targets for multiple sites

### DISCUSSION THREAD

**Discussion by ghost** [craftcms/cms#8698] [Ideas]
### Description

Additional preview targets are a great feature, but how to set it up for multi-language site with different target URIs?

For example you have a news archive section (single) set up like this:

![image](https://user-images.githubusercontent.com/25308802/59510244-7f725400-8eb3-11e9-8fe8-f33a17ccd565.png)

and want that to be a preview target for the news entries. That would require different target URL formats for every site.

Maybe you can work around with a complex URL format like `{{object.section.handle == 'en' ? 'news' : 'nachrichten'}}` (untested), but it would be better to have a third column specifying the sites (all/specific) where the preview target should be used.

### Additional info

- Craft version: 3.2 beta 2
- PHP version:
- Database driver & version:
- Plugins & versions:

**Comment by sarahschuetz** [comment]
@ghost did you figure out a solution?

**Reply by thupsi** [reply]
You can use something like `{{ craft.entries.section('newsIndex').one().uri }}` and it pulls the correct uri for each site. Or am I missunderstanding the question?
